lib/WorkerHandler.js

```diff
@@ -116,6 +129,12 @@ function WorkerHandler(script) {
     me.processing = {};
   }
 
+  // send all queued request to worker
+  function dispatchQueuedRequests()
+  {
+    me.request_queue = me.request_queue.filter(me.worker.send.bind(me.worker));
```

Comment:
This is more functional approach, but since it's not used elsewhere so much I'll change it to #forEach.